PL2/PL2-parser/src/main/java/nl/tudelft/pl2016gr2/parser/controller/FullGfaReader.java

```diff
@@ -151,15 +152,15 @@ private int parseNodeGenoms(Node node, char[] chars, int curIndex) {
    *
    * @param node     the node of which the orientation is read.
    * @param chars    the character array of the node line in the GFA file.
-   * @param curIndex the current index; where the genoms end in the line.
+   * @param curIndex the current index; where the genomes end in the line.
    */
   private void parseNodeOrientation(Node node, char[] chars, int curIndex) {
     int index = skipTillCharacter(chars, curIndex, '\t', 3);
     index = skipTillCharacter(chars, index, ':', 2) + 1;
     int orientation = 0;
     while (index < chars.length) {
-      orientation *= 10;
-      orientation += chars[index++] - 48;
+      orientation *= SHIFT_BY_BASE_10;
+      orientation += chars[index++] - '0';
     }
```

Comment:
See the comment 2 above this one. You might have to click "Show 2 comments", because github seems to hide comments which apply to lines which have been changed after the comment was made.